lib/trackler/implementation.rb

```diff
@@ -91,12 +89,19 @@ def readme_title
       problem.name
     end
 
+    def optional_blurb
+      blurb = problem.blurb
+      return '' if problem.description.start_with?(blurb)
+      "#{blurb}\n\n"
+    end
```

Comment:
What happens when we've removed the blurb from the metadata file, and only have it in the description?

We have places on the site where we need the blurb.